src/App.tsx

```diff
@@ -14,11 +15,100 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  selectedGood: string[],
+};
 
-export default App;
+type Props = {};
+
+export class App extends React.Component<Props, State> {
```

Comment:
you could write like this:
```suggestion
export class App extends React.Component<{}, State> {
```

Reply:
I agree with @maxxnikiforov . What if you add some props tomorrow? You will create type Props with this prop, but now you just should add this prop to the exit type.

Reply:
Согласен, знаю, что так можно записать тип Props